pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Portable Document Format Accessibility APIs Mapping (PDF-AAM) Task Force
+permalink: /about/groups/task-forces/pdf-aam/
+ref: /about/groups/task-forces/pdf-aam/
+lang: en
+github:
+  label: wai-groups
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Summary" class="" %}
+{:/}
+
+The Portable Document Format Accessibility APIs Mapping (PDF-AAM) Task Force is a task force of the [Accessible Rich Internet Applications (ARIA) Working Group](/about/groups/ariawg/).
+
+Some information on this page is also shown on, and may be more current in, the [PDF-AAM page](https://www.w3.org/groups/tf/pdf-aam/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2..3" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Objective
+
+The mission of the PDF-AAM Task Force is to specify how user agents should map accessibility semantics within the Portable Document Format specification. This Task Force provides a focused forum for this work while the Working Group continues its work on ARIA development.
+
+### Work Statement
+
+The [PDF-AAM Task Force Work Statement](/about/groups/task-forces/pdf-aam/work-statement/) defines the initial objective, scope, approach, and participation of the Task Force.
+
+## Meetings and Communication
+
+The PDF-AAM Task Force conducts its work using a variety of synchronous and asynchronous tools.
+
+- Teleconferences of the ARIA Working Group and its task forces (also see [meeting minutes](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes))
```

Comment:
Suggest removing the following as it is not so much a channel as a record, and because the information is duplicated further down the page.
(also see [meeting minutes](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes))

Reply:
Done